lib/Runtime/Language/SourceTextModuleRecord.cpp

```diff
@@ -623,6 +644,7 @@ namespace Js
             });
         }
 
+        ModuleNamespace::GetModuleNamespace(this);
         Js::AutoDynamicCodeReference dynamicFunctionReference(scriptContext);
```

Comment:
It looks like this initializes the namespace object immediately? Can we delay that until we try to use resolve it? It might be that no one uses the namespace object for this module.